action.yml

```
@@ -15,6 +15,10 @@ inputs:
   artifacts-path:
     description: 'The path to the directory where this action will create its artifact files.'
     required: false
+  create-draft-pr:
```

I think the most clear option would be to rename this as follows:
```suggestion
  created-pr-status:
```

We can then keep the Bash variables you created the same and parse this input to true/false, or we can rename the variables as well.